Makefile

```diff
@@ -422,7 +422,7 @@ unit-test: ## Run unit tests
 unit-test: $(SETUP_ENVTEST) 
 unit-test: KUBEBUILDER_ASSETS ?= $(shell $(SETUP_ENVTEST) use --use-env -p path --arch $(GO_ARCH) $(KUBEBUILDER_ENVTEST_KUBERNETES_VERSION))
 unit-test:
-	KUBEBUILDER_ASSETS="$(KUBEBUILDER_ASSETS)" $(GO_TEST) $(UNIT_TEST_PACKAGES) -cover -tags "$(BUILD_TAGS)" $(GO_TEST_FLAGS)
+	KUBEBUILDER_ASSETS="$(KUBEBUILDER_ASSETS)" $(GO_TEST) $(UNIT_TEST_PACKAGES) -cover -tags "$(BUILD_TAGS)" $(GO_TEST_FLAGS) -ldflags "-X github.com/aws/eks-anywhere/pkg/version.gitVersion=v0.0.0-dev"
```

Comment:
What is the purpose of this change? Is this something we want to commit into the repo, or a change made for testing purposes?

Reply:
This is needed for unit tests. Admittedly, the unit tests that use version.gitVersion are not in this PR but it will be needed in related PRs.

Reply:
We don't need it. I removed it.